integrationTests/chainSimulator/common.go

```diff
@@ -28,7 +36,37 @@ var (
 	InitialAmount = big.NewInt(0).Mul(OneEGLD, big.NewInt(100))
 )
 
-// GenerateTransaction will generate a transaction based on input data
+// GetSysAccBytesAddress will return the system account bytes address
+func GetSysAccBytesAddress(nodeHandler process.NodeHandler) ([]byte, error) {
```

Comment:
We could've made this receive t *testing.T so it doesn't return error anymore and therefore at caller level there is only one line of code to get this address